e2e_tests/test_admin.py

```diff
@@ -10,6 +10,11 @@ def admin_login(base_url: str, page: Page):
     page.get_by_text("Anmelden").click()
 
 
+def go_to_cap_edit_board(page: Page, city: str):
+    page.get_by_text("KAP editieren").click()
```

Comment:
This should be renamed to "Handlungsfeld editieren" right (in implementation too)? If so then all other "cap" mentions here should be replaced by englisch version of "Handlungsfeld" (if we already have some).

Reply:
I just thought because of the renaming in #32 "KAP bearbeiten" to "Handlungsfelder / Maßnahmen bearbeiten" but if you say that it makes no sense then resolve this conversation 🙂